bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gems/Gems.java

```diff
@@ -166,6 +166,9 @@ public double getAOCorrectedFWHM(boolean doThrow) {
                                 return 0.12;
                         }
                         break;
+
+                    default:
+                        throw new IllegalArgumentException("GeMS does not support band " + strehlBand);
```

Comment:
I don't really know the context here but since we're talking about J, H, K perhaps it should say GSAOI instead?  I'm not sure what "GeMS does not support band" means in general though.